cmd/zpool/Makefile.am

```diff
@@ -129,10 +129,56 @@ zpoolconfdefaults = \
 	test_progress \
 	test_ended
 
+zpoolcompatdir = $(pkgdatadir)/compatibility.d
+
+dist_zpoolcompat_DATA = \
+	compatibility.d/compat-2018 \
+	compatibility.d/compat-2019 \
+	compatibility.d/compat-2020 \
+	compatibility.d/compat-2021 \
+	compatibility.d/freebsd-11.0 \
+	compatibility.d/freebsd-11.2 \
+	compatibility.d/freebsd-11.3 \
+	compatibility.d/freenas-9.10.2 \
+	compatibility.d/grub2 \
+	compatibility.d/o3x-1.7.0 \
+	compatibility.d/o3x-1.8.1 \
+	compatibility.d/o3x-1.9.3 \
```

Comment:
Question for OS X users: is the o3x naming scheme intuitive?

Reply:
[Note: I'm not the PR submitter.] In context, I assume that must be **O**penZFS **o**n **O**S **X**, but I certainly would not have guessed that without the context. Also, the OS is not called OS X any more; it's now macOS. But at the same time, I'm not sure that e.g. macos-1.7.0 makes sense either, since 1.7.0 is not a macOS version. And "maczfs" is a different project. So I really don't have any better idea.

Reply:
cc: @lundman

Reply:
I see the term "o3x" used throughout the OpenZFSonOSX [wiki page](https://openzfsonosx.org/wiki/Main_Page), it seemed a little less cumbersome than "openzfsonosx" but on reflection the latter is clearer, will amend. @lundman WDYT?

Reply:
Honestly, I'm not sure myself.  o3x is certainly easier to use in shorthand, like zol. But do we use those terms in documentation? Then there is that whole OS X -> macOS thing. Dangit Apple, I have to pay for those domains!

Reply:
I think we can leave it as "openzfsonosx" for now; if you later change the project name we can add symlinks as needed.